### PR TITLE
UIDATIMP-22: Add "Jobs" pane layout.

### PR DESCRIPTION
**Pull request by VictorSoroka1**

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "singleQuote": true,
```
Comment (oleksii-popov) [inline .prettierrc]: Do we actually use Pretier?

File: src/index.js [hunk 2]
```diff
@@ -13,14 +13,24 @@ class DataImport extends React.Component {
     showSettings: PropTypes.bool,
   };
 
+  constructor(props) {
+    super(props);
+
```
Comment (oleksii-popov) [inline src/index.js]: please remove empty lines

File: src/index.js [hunk 3]
```diff
@@ -13,14 +13,24 @@ class DataImport extends React.Component {
...
       return <Settings {...this.props} />;
     }
 
     return (
       <Switch>
-        <Route path={`${this.props.match.path}`} exact component={Application} />
+        <Route
+          path={`${this.props.match.path}`}
+          exact
+          render={() => <this.connectedHome {...this.props} />}
```
Comment (oleksii-popov) [inline src/index.js]: PLease move to additional method and add comments.

File: src/index.js (hunk 2 above)
Comment (VictorSoroka1) [reply src/index.js]: I usually put an empty line after the `super`. So this one is not accidental.

File: src/index.js (hunk 3 above)
Comment (VictorSoroka1) [reply src/index.js]: Will do.

File: .prettierrc (hunk 1 above)
Comment (VictorSoroka1) [reply .prettierrc]: I personally use it for sure while development. It is up to the developer to enable it for the redactor/IDE.

**Pull request merged**